guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<ol><li>For New Zealand, the fieldCategory is very different, in terms of that it includes categories of merchandise expenditures, which are not seen in that of the U.S., Do you know what decides the field categories to be included in different Country objects?</li><li>What does "MBR" mean? Is there also "CBR" categories? Would be helpful if a link to help doc explaining these field categories is provided.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>For 1 - categories are based on GeoEnrichment coverage for the country. I have restated the link to data browser that shows what categories are covered.</p><p> </p><p>For 2 - MBR here is the data provider "<a href="https://doc.arcgis.com/en/esri-demographics/reference/data-providers.htm#ESRI_SECTION1_16BA150DDCBB43EAA21476D1210D3FB5" target="_blank">Michael Bauer Research</a>"</p>